docs/en/reference/basic-mapping.rst

```diff
@@ -200,36 +200,48 @@ mapping from a PHP string to a SQL VARCHAR (or VARCHAR2 etc.
 depending on the RDBMS brand). Here is a quick overview of the
 built-in mapping types:
 
--  ``string``: Type that maps a SQL VARCHAR to a PHP string.
--  ``integer``: Type that maps a SQL INT to a PHP integer.
--  ``smallint``: Type that maps a database SMALLINT to a PHP
+-  ``string``: Type that maps an SQL VARCHAR to a PHP string.
+-  ``integer``: Type that maps an SQL INT to a PHP integer.
+-  ``smallint``: Type that maps an database SMALLINT to a PHP
```

Comment:
That was probably from a proponent of the /ˈsiːkwəl/  pronunciation :)

Reply:
A decision should be made so we can have all types in the documentation.

Reply:
"a database", not "an database"

Reply:
```suggestion
-  ``smallint``: Type that maps a database SMALLINT to a PHP
```